src/go/rpk/pkg/cli/debug/bundle/bundle_k8s_linux.go

```diff
@@ -135,13 +137,28 @@ func executeK8SBundle(ctx context.Context, bp bundleParams) error {
 		if err != nil {
 			errs = multierror.Append(errs, err)
 		}
+		errs.ErrorFormat = errorFormat
 		fmt.Println(errs.Error())
 	}
 
 	fmt.Printf("Debug bundle saved to %q\n", f.Name())
 	return nil
 }
 
+// adminAddressesUnion returns the union of two slices of adminAddresses.
+func adminAddressesUnion(a, b []string) []string {
+	m := make(map[string]bool) // track unique addresses.
```

Comment:
`map[string]struct{}` is technically more efficient here. Though it's much less pleasant to use. This is more of a "the more you know" comment, no change needed IMO.